base/intrinsics/intrinsics.odin

```diff
@@ -353,15 +353,18 @@ x86_xgetbv :: proc(cx: u32) -> (eax, edx: u32) ---
 objc_object   :: struct{}
 objc_selector :: struct{}
 objc_class    :: struct{}
+objc_ivar     :: struct{}
+
 objc_id    :: ^objc_object
 objc_SEL   :: ^objc_selector
 objc_Class :: ^objc_class
+objc_Ivar  :: ^objc_ivar
 
 objc_find_selector     :: proc($name: string) -> objc_SEL   ---
 objc_register_selector :: proc($name: string) -> objc_SEL   ---
 objc_find_class        :: proc($name: string) -> objc_Class ---
 objc_register_class    :: proc($name: string) -> objc_Class ---
-
+ivar_get			   :: proc(self: ^$T, $U: typeid) -> ^U ---
```

Comment:
This should be prefixed with `objc_` imo

Reply:
I agree. I reverted to that already locally, but haven't pushed yet.

Reply:
It seems I was wrong. I had renamed it only in the docs. Fixed now.